### PR TITLE
Terraform example for RDS 

### PR DESCRIPTION
**Pull request by sathiyas**
Check README for more details on user/password and options are exposed in variables for postgres / mysql.

### CODE REVIEW

File: examples/aws-rds/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Creating an RDS insatnce in AWS
```
Comment (scalp42) [inline examples/aws-rds/README.md]: There's a typo here :eyeglasses:

File: examples/aws-rds/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+This example provides sample configuration for creating a mysql or postgres insatnce. For Oracle/SQL Servers, replace default values with appropriate values, they are not included in sample since the number of options are high.
```
Comment (scalp42) [inline examples/aws-rds/README.md]: There's a typo here: `insatnce` vs `instance`

File: examples/aws-rds/README.md (hunk 1 above)
Comment (sathiyas) [inline examples/aws-rds/README.md]: Thx, corrected

File: examples/aws-rds/variables.tf [hunk 3]
```diff
@@ -0,0 +1,41 @@
+variable "identifier" {
+	default = "mydb-rds"
+    description = "Idnetifier for your DB"
```
Comment (mikeys) [inline examples/aws-rds/variables.tf]: Idnetifier => Identifier

**Comment by sathiyas** [comment]
thx @mikeys

**Comment by mitchellh** [comment]
LGTM

**Pull request merged**